sencha-workspace/SlateTasksTeacher/app/controller/Tasks.js

```diff
@@ -363,6 +363,12 @@ Ext.define('SlateTasksTeacher.controller.Tasks', {
                 Section: section.getData()
             }, task || null));
 
+            formPanel.getParentTaskField().getStore().filter({
+                id: 'exclude-self',
```

Comment:
@nbey FYI, I found that the initial implementation of this would keep adding an additional filter each time you opened the window. The [FilterCollection](https://docs.sencha.com/extjs/6.2.1/classic/Ext.util.FilterCollection.html) instance supports keying members by `id`, so this implementation will ensure that the same filter is updated each time. It also serves to ensure that the filter is removed when you open the "create" version of the form after opening an edit version, as the value checked for won't be a real ID that matches anything in the Tasks store